ckan/config/deployment.ini_tmpl

```diff
@@ -191,6 +191,8 @@ ckan.hide_activity_from_users = %(ckan.site_id)s
 #smtp.password = your_password
 #smtp.mail_from =
 
+## RQ Job Settings
+rq.bg_job_timeout = 180
```

Comment:
IMO, we should choose a implementation-agnostic name for the configuration variable, for example `ckan.job_timeout`. What do you think, @amercader?

Reply:
Changed to `bg_job_timeout` and changed references to this config variable.